frontend/js/src/missing-mb-data/MissingMBData.tsx

```diff
@@ -198,14 +214,29 @@ export default class MissingMBDataPage extends React.Component<
                     )}
                   />
                 );
+                const additionalMenuItems = [
+                  <ListenControl
+                    text="Link with MusicBrainz"
```

Comment:
Reo pointed out in the IRC channel that any logged in user can link a listen from someone else's page.
IIRC the outcome of the discussion is that this shouldn't be an issue as there is a form of consensus on the user-provided mappings.

I just wanted to make sure we don't want to limit this to the currently-logged-in-user's own page.

Reply:
In any case, we should either not show this option or show it as disabled when no user is logged in (`!currentUser`)  since the modal will be missing (= non-working option with no feedback on click)

Reply:
As pointed out in LB-1198, it is fine to show the Link with MusicBrainz button on other people's profile. I'll fix the logged in thing.